sqlglot/dialects/snowflake.py

```diff
@@ -805,6 +806,13 @@ def _parse_show_snowflake(self, this: str) -> exp.Show:
                 },
             )
 
+        def _parse_put(self) -> exp.Put:
+            self._match(TokenType.PUT)
```

Comment:
I think this is redundant, we should've already consumed the token before invoking the statement parser.

Reply:
Thanks, removed 👍